### ISSUE TITLE
Wrong Tried to register two views with the same name RNFirebaseAdMobBanner

### ISSUE THREAD

**Issue by diennguyentien** [invertase/react-native-firebase#497]
ExceptionsManager.js:65 Invariant Violation: Tried to register two views with the same name RNFirebaseAdMobBanner
* for each sreen I register admob as below: 
```
const Banner = firebase.admob.Banner;
const AdRequest = firebase.admob.AdRequest;
const request = new AdRequest();
request.addKeyword('foobar');
...
render() {
  return (
    <Banner
      size={"LARGE_BANNER"}
      request={request.build()}
      onAdLoaded={() => {
        console.log('Advert loaded and is now visible');
      }}
    />
  );
}
```
I hope someonce can help

* Target Platform: only tested on android
* Development Operating System: MacOS
* Build Tools: Android Studio
* React Native version: 0.49.0
* RNFirebase Version: 3.0.2
* Firebase Module: Admob

**Comment by Ehesp** [comment]
When you say for each screen, are you using a navigator or are they on the same view?

**Comment by diennguyentien** [comment]
Hi @Ehesp, I using a navigator, I created a admob component, for each screen, I import and used it

**Comment by josesuero** [comment]
I'm having the same problem, have several ads on the same page used for internal ads

**Comment by kurtsergey** [comment]
I have similar problem. Several native ads on one screen do not work with react-native@0.49. It works well with react-native@0.48

**Comment by Ehesp** [comment]
I'll spin this up later and take a look.

**Comment by Ehesp** [comment]
Hi Guys, I think this was an issue introduced with fibre. Ive pushed up a fix locally where it only creates a single instance of the native view rather than one per JS component. I too was getting the same error when placing multiple on the screen.

https://github.com/invertase/react-native-firebase/tree/fix-497

Please could you install this branch (JS only change) and see if it works for your use-cases. Multiple now work on the test app:

![image](https://user-images.githubusercontent.com/842078/31836886-bf9ded4c-b5ce-11e7-94e9-58407a6379b2.png)

Thanks

**Comment by josesuero** [comment]
update works for me

**Comment by juanamd** [comment]
If this works correctly, could you please add the change to the 2.x branch as well? From what I've checked, the fix should be compatible there too.

**Comment by chrisbianca** [comment]
This has now been released as part of v3.0.5

@juanamd I can't guarantee this will make it back to the 2.x branch - it's not just a case of applying the changes, it's the time it takes performing testing, builds, etc.  There's only the 3 of us, and we're striving to make the library as good and resilient as we can off our own back.  We're doing this by moving forward and hope you'll upgrade to v3 - there's already such a vast amount of new features and fixes that you'll really feel the benefit.

**Comment by juanamd** [comment]
I hear you, Chris.
I have applied the changes locally, and everything seems to work fine. I'll handle with that until I make the move to 3.x, when I have the time to test everything properly.
Thanks!
